sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -34,9 +35,11 @@ export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResul
 }
 
 // @public
-export class CognitiveServicesCredential extends ApiKeyCredentials {
+export class CognitiveServicesCredential implements ServiceClientCredentials {
     constructor(subscriptionKey: string);
-}
+    setSubscriptionKey(subscriptionKey: string): void;
+    signRequest(webResource: WebResource): Promise<WebResource>;
```

Comment:
What does this do?

Reply:
This is how the class is able to modify the outgoing request to add the auth headers.

Reply:
This kind of stuck out to me in the API md file too. In the old MD file it was just rolled into the `extends ApiKeyCredentials` but now we're seeing it because after the change it's no longer inherited from the type in `core-http`.